src/coord/rmr/conn.c

```diff
@@ -524,13 +524,12 @@ static int checkTLS(char** client_key, char** client_cert, char** ca_cert, char*
   char* clusterTls = NULL;
   char* tlsPort = NULL;
 
+  // If `tls-cluster` is not set to `yes`, we do not connect to the other nodes
+  // with TLS.
   clusterTls = getRedisConfigValue(ctx, "tls-cluster");
   if (!clusterTls || strcmp(clusterTls, "yes")) {
```

Comment:
Necessarily lowercase? Should we switch to `strcasecmp`?

Reply:
I'm quite sure Redis works only with lowercase values